fabric-keybindings-v0/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,31 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-keybindings-v0",
+  "name": "Fabric Key Bindings (v0)",
+  "version": "${version}",
+  "environment": "client",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-keybindings-v0/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.4.0",
+    "fabric-api-base": "*"
+  },
+  "description": "Keybinding registry API.",
+  "mixins": [
+    "fabric-keybindings-v0.mixins.json"
+  ],
+  "entrypoints": {
+    "main": [
+      "net.fabricmc.fabric.impl.client.keybinding.TestMod"
```

Comment:
Leave the test mods out for now, unless if you copy the official fabric buildscript for actual testmod support